app/server/skychat/DatabaseHelper.ts

```diff
@@ -12,6 +12,6 @@ export class DatabaseHelper {
     static db: sqlite.Database;
 
     static async load(): Promise<void> {
-        DatabaseHelper.db = await sqlite.open('./database.db');
+        DatabaseHelper.db = await sqlite.open('./database/main.db');
```

Comment:
Database moved to `database` folder so that it can be easily mounted to a docker image